episodes/computing-pi.md

```diff
@@ -94,10 +93,10 @@ def calc_pi_numpy(N):
 
 This is a **vectorized** version of the original algorithm. It nicely demonstrates **data parallelization**,
 where a **single operation** is replicated over collections of data.
-It contrasts to **task parallelization**, where **different independent** procedures are performed in
-parallel (think for example about cutting the vegetables while simmering the split peas).
+It contrasts with **task parallelization**, where **different independent** procedures are performed in
```

Comment:
This is a horrible paragraph (my own fault!). Maybe better:

This is a **vectorized** version of the original algorithm. Once a problem is written in a vectorized form it becomes amenable to **data parallelization**. Each single operation is replicated over a large collection of data. Data parallelism contrasts to **task parallelism**, where different independent procedures are performed in parallel (an example of task parallelism is the pea-soup recipe in the introduction).